src/applications/edu-benefits/5490/helpers.jsx

```diff
@@ -84,3 +102,30 @@ export const benefitsDisclaimerChild = (
     .
   </p>
 );
+
+export const ageWarning = (
+  <div
+    className="vads-u-display--flex vads-u-align-items--flex-start vads-u-background-color--primary-alt-lightest vads-u-margin-top--3 vads-u-padding-right--3"
+    aria-live="polite"
+  >
+    <div className="vads-u-flex--1 vads-u-margin-top--2p5 vads-u-margin-x--2 ">
+      <i className="fas fa-info-circle" />
+    </div>
+    <div className="vads-u-flex--5">
+      <p className="vads-u-font-size--base">
+        Applicants under the age of 18 cannot legally make a benefits election.
+        Based on your date of birth, please have a parent, guardian or custodian
+        review the information on this application and click the “Submit
+        application” button at the end of this form.
+      </p>
+    </div>
+  </div>
+);
+
+export const eighteenOrOver = birthday => {
+  return (
+    birthday === undefined ||
+    moment().diff(moment(birthday, 'YYYY-MM-DD'), 'years') > 17 ||
+    birthday.length !== 10
```

Comment:
Should probably check valid date length prior to checking years

Reply:
Probably a good idea there. I will switch them around.